tests/brokers/kafka/test_consume.py

```diff
@@ -163,3 +165,67 @@ async def handler(msg: KafkaMessage):
             assert not m.mock.called
 
         assert event.is_set()
+
+    @pytest.mark.asyncio
+    @pytest.mark.slow
+    async def test_consume_validate_false(
```

Comment:
Please, move this testcase from final broker case to basic TestCase class: it has no broker-based dependencies, so we can write it at once and need no create a copies